metrics/go_metrics_test.go

```diff
@@ -790,6 +790,46 @@ func TestRecordRequestPrivacy(t *testing.T) {
 	assert.Equal(t, m.PrivacyTCFRequestVersion[TCFVersionV2].Count(), int64(1), "TCF V2")
 }
 
+func TestRecordAdapterBuyerUIDScrubbed(t *testing.T) {
+	var fakeBidder openrtb_ext.BidderName = "fooAdvertising"
+	adapter := "AnyName"
+	lowerCaseAdapterName := "anyname"
+
+	tests := []struct {
+		description     string
+		metricsDisabled bool
+		adapterName     openrtb_ext.BidderName
+		expectedCount   int64
+	}{
+		{
+			description:     "",
+			metricsDisabled: false,
+			adapterName:     openrtb_ext.BidderName(adapter),
+			expectedCount:   1,
+		},
+		{
+			description:     "",
+			metricsDisabled: false,
+			adapterName:     fakeBidder,
+			expectedCount:   0,
+		},
+		{
+			description:     "",
+			metricsDisabled: true,
+			adapterName:     openrtb_ext.BidderName(adapter),
+			expectedCount:   0,
```

Comment:
Could you add descriptions for these tests?

Reply:
Added missing descriptions for this test and an old test that I used as a template for this. I also converted both to use `t.Run`.